### PR TITLE
GCP marketplace

### PR DESCRIPTION
**Pull request by kelseiv**
@gunnaraasen, think I got everything we discussed. For GCP Cloud, I referenced the doc. Please verify updates. Thanks!

@gunnaraasen, here's draft re: https://github.com/influxdata/docs.influxdata.com/issues/2743

- updated menu names so GCP and AWS align a bit (not sure if we need separate GCP configurations and Deployment page?)
- updated menu so GCP is visible in navigation bar
- removed BYOL refs and updated content as applicable
- verified all links work (please verify if still needed)
- hid graphics - we can probably remove these screenshots (like we did for AWS) given we're linking directly to GCP site
- added numbered steps; adjusted alignment

.............................................................................

 PS When I attempted to sign up for free trial on the portal.influxdata.com/licenses page, and click (Please copy the license key below and head back to the **GCP Marketplace** to deploy your InfluxDB Enterprise trial.)

Got this error:
You do not have sufficient permissions to view this page.
Tracking Number: 2443986112183562438

### CODE REVIEW

File: content/enterprise_influxdb/v1.7/install-and-deploy/google-cloud-platform.md [hunk 1]
```diff
@@ -4,82 +4,94 @@ aliases:
...
-> **Note:** The [Deployment Manager templates](https://cloud.google.com/deployment-manager/) used for the InfluxDB Enterprise BYOL solution are [open source](https://github.com/influxdata/google-deployment-manager-influxdb-enterprise). Issues and feature requests for the Marketplace deployment should be [submitted through the related GitHub repository](https://github.com/influxdata/google-deployment-manager-influxdb-enterprise/issues/new) (requires a GitHub account) or by contacting [InfluxData support](mailto:Support@InfluxData.com).
+1. [Verify prerequistes](#verify-prerequisites).
+2. [Deploy a cluster](#deploy-a-cluster).
+3. [Access the cluster](#access-the-cluster).
 
-## Prerequisites
+After deploying your cluster, see [Getting started with InfluxDB](/platform/introduction/getting-started) for an introduction to InfluxDB database and the InfluxData platform.
 
-This guide requires the following:
+>**Note:** Want to deploy your InfluxDB cluster on the Cloud? Check out [InfluxDB Cloud 2.0](/https://v2.docs.influxdata.com/v2.0/cloud/get-started/).
```
[the remark this answers is not in the document]
Comment (kelseiv) [reply content/enterprise_influxdb/v1.7/install-and-deploy/google-cloud-platform.md:20]: excellent!

File: content/enterprise_influxdb/v1.7/install-and-deploy/google-cloud-platform.md [hunk 2]
```diff
@@ -4,82 +4,94 @@ aliases:
...
 
-![GCP InfluxDB Enterprise deployment pending page](/img/enterprise/gcp/byol-intro-3.png)
+  > **Note:** Typically, fields in collapsed sections don't need to be altered.
 
-The cluster will take up to five minutes to fully deploy. If the deployment does not complete or reports an error, read through the list of [common deployment errors](https://cloud.google.com/marketplace/docs/troubleshooting).
+5. Adjust meta node fields as needed, including:
 
-![GCP InfluxDB Enterprise deployment complete page](/img/enterprise/gcp/byol-intro-4.png)
+  - Meta node instance count: Enter the number of meta nodes to include in your cluster (we recommend using the default, 3, in most cases).
+  - Meta node machine type: Select the virtual machine type to use for meta nodes (by default, 1 vCPUs). Use the down arrow to scroll through list. Notice the amount of memory available for the selected machine. To alter the number of cores and memory for your selected machine type, click the **Customize** link.
```
[the remark this answers is not in the document]
Comment (kelseiv) [reply content/enterprise_influxdb/v1.7/install-and-deploy/google-cloud-platform.md:57]: no, I think that's a great idea

File: content/enterprise_influxdb/v1.7/install-and-deploy/google-cloud-platform.md [hunk 3]
```diff
@@ -4,82 +4,96 @@ aliases:
...
+>**Note:** InfluxDB Enterprise on GCP is a self-managed product. For a fully managed InfluxDB experience, check out [InfluxDB Cloud 2.0](/https://v2.docs.influxdata.com/v2.0/cloud/get-started/).
 
-- A [Google Cloud Platform (GCP)](https://cloud.google.com/) account with access to the [GCP Marketplace](https://cloud.google.com/marketplace/).
+## Verify prerequisites
+
+Before deploying an InfluxDB Enterprise cluster on GCP, verify you have the following prerequisites:
+
+- A [Google Cloud Platform (GCP)](https://cloud.google.com/) account with access to the [GCP Marketplace](https://cloud.google.com/marketplace/).	Before deploying an InfluxDB Enterprise cluster on GCP, verify you have the following prerequisites:
+
+- A [GCP](https://console.cloud.google.com/) account with access to the [GCP Marketplace](https://console.cloud.google.com/marketplace).
```
Comment (gunnaraasen) [inline content/enterprise_influxdb/v1.7/install-and-deploy/google-cloud-platform.md:28]: This bullet is still duplicated

Comment (kelseiv) [reply content/enterprise_influxdb/v1.7/install-and-deploy/google-cloud-platform.md]: egads. I read your edit incorrectly, thanks @gunnaraasen

**Pull request merged**